### PR TITLE
Fe-Feat/PokeTradeDetails Page Updates & Trade page update

### PR DESCRIPTION
**Pull request by Dalton-Klein**
Backend Updates:
  -added columns to userData Table for transaction histories

PokeTradeDetails Page:
  -Added dynamic values for number of sales
  -Seller Rating is dynamic
  -Seller num ratings dynamic

Trade Page:
  -MobileFirst with that many components is difficult, we can still change layout of page, this is DRAFT
  -Reworked design to be mobile first
  -Only need one text input, set of buttons (two users will not be sharing one keyboard)

### CODE REVIEW

File: client/trademon/src/components/aaPageComponents/landingPage.tsx [hunk 1]
```diff
@@ -22,8 +22,8 @@ export default function LandingPage() {
         <SearchBar></SearchBar>
       </div>
       <div className="platforms-container">
-        {worlds.map((world) => (
-          <PlatformContainer key={0} world={world}></PlatformContainer>
+        {worlds.map((world, index) => (
```
Comment (obi-wil) [inline client/trademon/src/components/aaPageComponents/landingPage.tsx:25]: Good. I saw it yesterday but forgot to fix it...

File: client/trademon/src/components/tileComponents/myProfileTileComponents/profileBannerComponent.tsx [hunk 2]
```diff
@@ -16,19 +16,13 @@ export default function ProfileBanner() {
   }, []);
 
   const setSellerRating = () => {
-    let totalRating: number = 0;
-    userData.sellerRating.forEach((rating: number) => {
-      totalRating += rating;
-    });
-    setSellerRatingValue(totalRating/userData.sellerRating.length);    
+    let totalRating: number = calcRating(userData.sellerRating);
```
Comment (obi-wil) [inline client/trademon/src/components/tileComponents/myProfileTileComponents/profileBannerComponent.tsx:19]: Neat. Helper function making 5 lines into 2

Comment (Dalton-Klein) [reply client/trademon/src/components/tileComponents/myProfileTileComponents/profileBannerComponent.tsx:19]: Its cause I use that same function in a few places on different files (:

File: client/trademon/src/utils/helperFunctions.ts [hunk 3]
```diff
@@ -8,4 +8,13 @@ export default function setAppraisalImage(appraisal: number| undefined)  {
...
+}
+
+export const calcRating = (array: number[]) => {
+  let totalRating: number = 0;
+  array.forEach((rating: number) => {
+    totalRating += rating;
+  })
+  const result = totalRating/array.length
+  return result;
 }
```
Comment (obi-wil) [inline client/trademon/src/utils/helperFunctions.ts:20]: Is prettier off? Missing newline here

File: client/trademon/src/utils/interfaces.ts [hunk 4]
```diff
@@ -3,6 +3,7 @@
 export interface Trades {
   id:number,
   numViews:number, 
+  UserDatumId:number,
```
Comment (obi-wil) [inline client/trademon/src/utils/interfaces.ts:6]: Latin... Yeah

**Review by gsjuradog — APPROVED** [review]
Lots of work, great

**Review by obi-wil — APPROVED** [review]
Looks good. Helper functions help!

**Review by DanB-Web — APPROVED** [review]

**Pull request merged**